public/os-files/indexdb-driver.js

```
@@ -0,0 +1,415 @@
+            // Potentially rethrow or handle so subsequent operations know init failed
+            throw error;
+        }
+    }
+
+    async function _updateParentDirectoryListing(parentPath, itemName, action) {
+        if (parentPath === itemName) return; // Cannot modify self in parent listing (e.g. root)
+
+        try {
+            const parentContentRecord = await getRecord(CONTENTS_STORE_NAME, parentPath);
+            let entries = [];
+            if (parentContentRecord && parentContentRecord.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)